codexctl/sync.py

```diff
@@ -130,6 +131,13 @@ def download(self, document, location="", overwrite=False):
                 self.logger.debug("Not overwriting file")
                 return True
 
+            if os.path.isfile(fileLocation) and incremental:
+                if not self.__is_newer(document, fileLocation):
+                    self.logger.debug(
+                        "Local file already exists and is newer, skipping"
+                    )
+                    return True

```

Comment:
```suggestion
            if os.path.isfile(fileLocation) and incremental and not self.__is_newer(document, fileLocation):
                self.logger.debug(
                     "Local file already exists and is newer, skipping"
                )
                return True
```
You'll need to let the autoformatter run after this change too, as I'm on mobile so I can't tell where the line breakpoints should be etc.

Reply:
Rewrote it a bit, extracting `os.path.isfile()`.